src/Infrastructure/Validator/Manifest/TaskManifestConfigTreeBuilderFactory.php

```diff
@@ -0,0 +1,297 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Validator\Manifest;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\ManifestConfigTreeBuilderFactoryInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+class TaskManifestConfigTreeBuilderFactory implements ManifestConfigTreeBuilderFactoryInterface
+{
+    /**
+     * @var string
+     */
+    public const NAME = 'task';
+
+    /**
+     * @var \SprykerSdk\Sdk\Infrastructure\Validator\Manifest\ManifestEntriesValidator
+     */
+    protected ManifestEntriesValidator $validationHelper;
+
+    /**
+     * @param \SprykerSdk\Sdk\Infrastructure\Validator\Manifest\ManifestEntriesValidator $validationHelper
+     */
+    public function __construct(ManifestEntriesValidator $validationHelper)
+    {
+        $this->validationHelper = $validationHelper;
+    }
+
+    /**
+     * Returns manifest name.
+     *
+     * @return string
+     */
+    public function getName(): string
+    {
+        return static::NAME;
+    }
+
+    /**
+     * @param array $config
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\TreeBuilder
+     */
+    public function getConfigTreeBuilder(array $config): TreeBuilder
```

Comment:
Looks weird, man. You actually build a configuration here. Why its in the factory not in the builder? Why method name `getConfigTreeBuilder` not `buildConfiguration`?

Reply:
it was renamed with @sergeyspryker  but probably the class name should be `TaskManifestConfiguration`. with method name everything ok. it's return `TreeBuilder`.